### DISCUSSION TITLE
Ability to run more languages from within? jupyter notebook integration?

### DISCUSSION THREAD

**Discussion by KernelBypass** [TriliumNext/Trilium#3727] [Ideas] [answered]
So, I am looking for a knowledge management system.
I was wondering if JavaScript is the only language currently supported to run from within? If so, are there plans to add support for other languages?

E.g., Python - maybe via Jupyter Notebook integration since JN is also open source.

**Comment by zadam** [comment, marked as answer]
Hi, no, not really. Supporting JS is super cheap since it's running both frontend / backend app. Adding support for anything else would mean much more trouble (for unclear benefit).
